blocks/image/package.json

```diff
@@ -23,6 +23,7 @@
     "twind": "0.16.17"
   },
   "devDependencies": {
+    "@blockprotocol/hook": "0.0.8",
```

Comment:
This should be in `dependencies` (my omission)

Reply:
I guess we are using only typings from this module.